ci/plugins/mzcompose/hooks/post-command

```diff
@@ -17,6 +17,9 @@ run() {
     bin/ci-builder run stable bin/mzcompose --mz-quiet --find "$BUILDKITE_PLUGIN_MZCOMPOSE_COMPOSITION" "$@"
 }
 
+# Run before potential "run down" in coverage
+docker inspect "$(docker ps --all --quiet)" > docker-inspect.log
```

Comment:
Nope, unfortunately this does not seem to work:

```
pstoev@Ubuntu-2004-focal-64-minimal:~$ docker inspect "$(docker ps --all --quiet)"
[]
Error: No such object: 4078075218ba
5a40d0ca202f
a053420c2324
29185fc20741
9c1b303d7489
fb583dd81895
00d9ca9c1807
```

I think you need `xargs` or a similar tool to convert the list of containers sparated by newlines to a list of containers separated by spaces.

Reply:
Oh right, I had it without `"` before and then it worked, but shellcheck mislead me. xargs is fine, will do that.